Gemfile

```diff
@@ -28,3 +28,4 @@ platforms :jruby do
 end
 
 eval File.read('Gemfile-custom') if File.exist?('Gemfile-custom')
+gem 'allocation_stats'
```

Comment:
The build fails because of this, prehaps it's better off in a platform directive or even just in `Gemfile-custom` as we've done similarly before for benchmarking extras (like ips stuff) before :)